src/context/body.ts

```diff
@@ -1,11 +1,13 @@
-import { ResponseTransformer } from '../response'
+import { ResponseTransformer, MockedResponse } from '../response'
 
 /**
  * Sets the body of the response without any `Content-Type` header.
  * @example
- * res(body('foo'))
+ * res(body('message'))
  */
-export const body = <T>(value: T): ResponseTransformer => {
+export const body = <BodyType extends string>(
```

Comment:
why this? this is done by `text` util and not `body` right?

Reply:
You are right! I've mistyped this generic and have just pushed the fix. `body` will extend text, but also `BufferSource`, `FormData`, and a few other types according to [this spec](https://developer.mozilla.org/en-US/docs/Web/API/Response/Response).

Reply:
Yes that's right 🚀